### ISSUE TITLE
CLI Tools Bundle for iOS 11

### ISSUE THREAD

**Issue by MCApollo** [mologie/blog#1]
This is a terrible way to contact someone, sorry about that. 
Your Twitter is inactive and I was scared your email was the same.

I’m compiling tmux to put on a APT/Cydia repo and hit a brick wall - *compat/forkpty-darwin.c', needed by 'compat/forkpty-darwin.o.*
Do you still have your edits that were used to compiled tmux in the binpack you released?

**Comment by mologie** [comment]
Hi, GitHub tickets go straight to my mailbox, so there's that thing. ;) I'm not using Twitter much at all, but people DM me there occasionally (which is way more terrible than GitHub tickets!)

I've compiled tmux back then with Xcode 7 and had modified my iOS usr/include directory a lot. I recall that it involved lots of copying of files from the macOS SDK into the iOS SDK, because especially the tty headers were absent completely. Some files from libkern/ need iOS-specific touches to support the arm architecture, which may or may not be relevant for tmux.

Unfortunately I don't have a copy of the modifications I made to my Xcode 7 installation anymore, so if you wish to compile tmux from source for iOS you'll need to wrangle yourself through the compiler error messages and patch up the SDK by copying in macOS SDK headers and possibly XNU kernel headers until it compiles (and runs.)
